### ISSUE TITLE
It wont let me download

### ISSUE THREAD

**Issue by RobloxNitroProdigyhacker** [cheat-engine/cheat-engine#2967]
Everytime i click download it opens a new tab with the name https://d1vdn3r1396bak.cloudfront net/installer/19202890/824465484901750 and it really is annoying i have already checked out my internet and it is fine so i need some help here

**Comment by fade2gray** [comment]
> https: //d1vdn3r1396bak. cloudfront net/installer/19202890/824465484901750

That's an incorrectly formatted URL - the correctly formatted URL works fine. Are you downloading from the official CE site?

**Comment by Banaanae** [comment]
That url is where the installer is stored, are you complaining that when you press download it loads the link which has the installer?

**Comment by RobloxNitroProdigyhacker** [comment]
> > https: //d1vdn3r1396bak. cloudfront net/installer/19202890/824465484901750
> 
> That's an incorrectly formatted URL - the correctly formatted URL works fine. Are you downloading from the official CE site?

yeah cheat engine . org?

**Comment by RobloxNitroProdigyhacker** [comment]
> That url is where the installer is stored, are you complaining that when you press download it loads the link which has the installer?

i know thats like the installer link but the thing is it loads in saying site cant be reached or whatever

**Comment by RobloxNitroProdigyhacker** [comment]
> > https: //d1vdn3r1396bak. cloudfront net/installer/19202890/824465484901750
> 
> That's an incorrectly formatted URL - the correctly formatted URL works fine. Are you downloading from the official CE site?
 
yes i know i made the url like that just so people dont click on the url

**Comment by RobloxNitroProdigyhacker** [comment]
i can take a screenshot an attach it on here

**Comment by cheat-engine** [comment]
ask your internet provider for access to that site or use a vpn servewrvice (e.,g proton vpn)
